ctw/standard/nyxis/map.xml

```diff
@@ -47,12 +46,12 @@
         <effect duration="1.5">slow digging</effect>
     </kit>
     <kit id="black-kit" parent="spawn-kit">
-        <item slot="6" material="stained glass" damage="7" amount="64"/>
+        <item slot="6" material="stained glass" damage="7" amount="48"/>
```

Comment:
You could merge the stained glass blocks from both kits into the main kit with `team-color="true"` instead of using damage values.

Reply:
I added the stained glass as separate kits for a fairly minor reason which is so black spawns with gray stained glass instead of light gray. With team-color=true black will spawn with light gray stained glass.

Reply:
You can set a `dye-color="gray"` in the `<team>` definition for the black team and it should set this. The potential downside is that armour colour will also change, so this won't help to solve this unless you are okay with that too.

Reply:
Since I changed the stained glass to sea lanterns, I don't think this is relevant anymore, but good to know that `dye-color` exists. That might save me some time with any future maps. I'm also pretty adamant about having black's armor color be black and the chat color gray, so I'll keep the separated armor kits.

(Also the reason for the chat color is because otherwise it'll be hard to spot the nametags if they were black.)